### PR TITLE
Adds wigs

### PR DESCRIPTION
**Pull request by AnturK**
:cl:
add: Wigs are now available in AutoDrobe
/:cl:

### CODE REVIEW

**Review by ghost — DISMISSED** [review]
watever

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -1010,7 +1010,8 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 					/obj/item/clothing/head/cueball = 1,
 					/obj/item/clothing/under/scratch = 1,
         			/obj/item/clothing/under/sailor = 1,
-        			/obj/item/clothing/ears/headphones = 2)
+        			/obj/item/clothing/ears/headphones = 2,
```
Comment (lzimann) [inline code/game/machinery/vending.dm]: This indentation is offending me

Comment (Jordie0608) [reply code/game/machinery/vending.dm]: it's not staggered in the actual file; just preview.

Comment (AnturK) [reply code/game/machinery/vending.dm]: It's probably spaces instead of tabs, i'll normalize this.

Comment (Jordie0608) [reply code/game/machinery/vending.dm]: Better yet it's both.

File: code/modules/clothing/head/misc_special.dm [hunk 2]
```diff
@@ -177,3 +177,45 @@
...
+
+
+/obj/item/clothing/head/wig
+	name = "wig"
+	desc = "A bunch of hair without a head attached."
+	icon_state = ""
+	item_state = "pwig"
+	var/hair_style = "Very Long Hair"
+	var/hair_color = "#000"
+	flags_inv = HIDEHAIR
```
Comment (lzimann) [inline code/modules/clothing/head/misc_special.dm]: already defined vars before any new declaration please

File: code/game/machinery/vending.dm (hunk 1 above)
Comment (Supermichael777) [reply code/game/machinery/vending.dm]: Find who did this and shoot them

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**